components/Movie.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { TouchableOpacity, Alert } from 'react-native'
+import styled from 'styled-components/native'
+
+import MovieDetails from './MovieDetails'
+
+const Movie = ({ id, title, poster_path, rating, overview}) => {
+  const onPress = () => {
+    Alert.alert(
+      `${title}`,
+      `Rating: ${rating} - ${overview}`,
+      [
+        { text: "Back to movies", onPress: () => console.log("Cancel Pressed")}
+      ]
+    );
+  }
+
+return (
+  <>
```

Comment:
you are returning only TouchableOpacity here so why you are using extra fragments here??